src/coreservices.cpp

```diff
@@ -377,6 +380,10 @@ void CoreServices::initialize(QApplication* pApp) {
     // the uninitialized singleton instance!
     m_pPlayerManager->bindToLibrary(m_pLibrary.get());
 
+#ifdef __MACOS_MEDIAPLAYER__
+    m_pMacOSMediaPlayerService = std::make_shared<MacOSMediaPlayerService>(*m_pPlayerManager);
+#endif
```

Comment:
This means that this feature will always be enabled for user (since I assume we will probably ship a Mixxx release with `MACOS_MEDIAPLAYER=on`), however I can imagine some user not wanting this integration to be on, so would you consider adding a user setting for this? Note that I'm not a Mac user, so happy to ignore that one!

Reply:
Good idea, I'm not sure where to put that though. Ideally, this should probably go into `DlgPrefMetadata` as proposed in https://github.com/mixxxdj/mixxx/pull/3483/files#diff-25906de66775af0b1495dd1a528cc4256dd1de4778a23a494db27fbb4c164a76

Reply:
Yeah I think it sounds sensible to unify it with #3483 !